src/hashtable.c

```diff
@@ -979,6 +986,24 @@ static void prefetchNextBucketEntries(iter *iter, bucket *current_bucket) {
     }
 }
 
+/* Prefetches the values associated with filled entries in the given bucket. */
```

Comment:
"Filled" is about positions that may contain an entry or be empty. A position is filled. An entry is always in a filled position.

We can refer to the callback in this comment too.

```suggestion
/* Prefetches the values associated with the entries in the given bucket by
 * calling the entryPrefetchValue callback in the hashtableType, if defined. */
```